### ISSUE TITLE
Add chalk to dependencies in @pgtyped/query

### ISSUE THREAD

**Issue by Maxpain177** [adelsz/pgtyped#105]
I've installed `@pgtyped/query` as the main dependency and `@pgtyped/cli` as the dev dependency and get an error:

```
Error: Cannot find module 'chalk'
Require stack:
- /app/node_modules/@pgtyped/query/lib/loader/sql/logger.js
- /app/node_modules/@pgtyped/query/lib/loader/sql/index.js
- /app/node_modules/@pgtyped/query/lib/preprocessor.js
- /app/node_modules/@pgtyped/query/lib/index.js
- /app/utils/graphql/context.js
- /app/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1029:15)
    at Function.Module._load (internal/modules/cjs/loader.js:898:27)
    at Module.require (internal/modules/cjs/loader.js:1089:19)
    at require (internal/modules/cjs/helpers.js:73:18)
    at Object.<anonymous> (/app/node_modules/@pgtyped/query/lib/loader/sql/logger.js:6:33)
    at Module._compile (internal/modules/cjs/loader.js:1200:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1220:10)
    at Module.load (internal/modules/cjs/loader.js:1049:32)
    at Function.Module._load (internal/modules/cjs/loader.js:937:14)
    at Module.require (internal/modules/cjs/loader.js:1089:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/app/node_modules/@pgtyped/query/lib/loader/sql/logger.js',
    '/app/node_modules/@pgtyped/query/lib/loader/sql/index.js',
    '/app/node_modules/@pgtyped/query/lib/preprocessor.js',
    '/app/node_modules/@pgtyped/query/lib/index.js',
    '/app/utils/graphql/context.js',
    '/app/index.js'
  ]
}
```

**Comment by steelbrain** [comment]
Why was this closed? I ran into this bug and had to add `chalk` to root project dependencies

IMO A deep dependency should be declarative about it's dependencies instead of expecting the root project to take care of it.

**Comment by Maxpain177** [comment]
@steelbrain see https://github.com/adelsz/pgtyped/issues/91

**Comment by adelsz** [comment]
@steelbrain `chalk` dep issues fixed in #114
